modules/richards/tests/gravity_head_2/tests

```diff
@@ -171,4 +171,43 @@
     rel_err = 1e-5
     abs_zero = 1e-4
   [../]
+
+  [./except01]
+    type = 'RunException'
+    input = 'gh01.i'
+    cli_args = 'Materials/rock/viscosity="1"'
+    expect_err = 'There are 2 Richards fluid variables, so you need to specify this number of viscosities, relperm_UO, seff_UO, sat_UO, density_UO, SUPG_UO'
+  [../]
+  [./except02]
+    type = 'RunException'
+    input = 'gh01.i'
```

Comment:
It looks like you didn't get them all. All of the tests below are using the same input file and `Outputs/file_base` so they'd potentially stomp on each other. 

For these exception tests, you might just consider removing the Outputs block altogether instead of trying to write several unique files.